CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 Unreleased changes.
+- [#1304](https://github.com/Zilliqa/zq2/pull/1304): Implement the following APIs for querying DS blocks:  `eth_getGasPrice`, `GetMinerInfo`, `GetNodeType`, `GetNumDSBlocks`, `GetNumPeers`, `GetNumTransactions`, `GetNumTxnsDSEpoch`, `GetNumTxnsTxEpoch`, `GetRecentTransactions`, `GetSmartContractSubState`, `GetSoftConfirmedTransaction`, `GetStateProof`, `GetTotalCoinSupply`, `GetTotalCoinSupplyAsInt`, `GetTransactionStatus`, `GetTransactionsForTxBlockEx`, `GetTxBlockRate`, `GetTxRate`, `GetTxnBodiesForTxBlockEx`, `TxBlockListing`
```

Comment:
Isn't this the list of APIs that you didn't implement in #1304 ? :stuck_out_tongue:

Reply:
Yep, copy pasted the wrong thing, fixed now